packages/core/src/fhircast/test-utils.test.ts

```diff
@@ -0,0 +1,34 @@
+import { FhircastEventContext } from '.';
+import { createFhircastMessageContext } from './test-utils';
+
+describe('FHIRcast Test Utils', () => {
+  describe('createFhircastMessageContext', () => {
+    test('Valid inputs', () => {
+      expect(createFhircastMessageContext('patient', 'patient-123')).toEqual<FhircastEventContext>({
+        key: 'patient',
+        resource: {
+          resourceType: 'Patient',
+          id: 'patient-123',
+        },
+      });
+      expect(createFhircastMessageContext('imagingstudy', 'imagingstudy-456')).toEqual<FhircastEventContext>({
+        key: 'study',
+        resource: {
+          resourceType: 'ImagingStudy',
+          id: 'imagingstudy-456',
+        },
+      });
+    });
+
+    test('Invalid inputs', () => {
+      // @ts-expect-error Invalid resource type, must be one a FHIRcast resource type
```

Comment:
Yes otherwise there will be a type error. This is kind of the new recommended way to deal with intentional invalid types, because when there is no longer a TS error, the comments themselves will cause a TS error and so you can be sure that the comments are actually necessary over the course of time (including when type definitions change).